presto-main/src/main/java/io/prestosql/dispatcher/DecoratingListeningExecutorService.java

```diff
@@ -144,8 +144,22 @@ public boolean awaitTermination(long timeout, TimeUnit unit)
 
     public interface TaskDecorator
     {
+        /**
+         * Receives an object an modifies its metadata.
```

Comment:
Do you think we should explain better what is this "object"?

This description doesn't help me understanding the modifications and what is this Runnable command object

Reply:
explanation added